### ISSUE TITLE
Color alias

### ISSUE THREAD

**Issue by djornada** [acrisci/i3-style#23]
Hi!
is possible to suport color alias, something like this:

...
set $color #455A64
...
background $color

**Comment by acrisci** [comment]
What is the use case for this?

**Comment by okraits** [comment]
I think i3-style already supports what you mean - take a look at the solarized theme bundled with i3-style for an example. If you rather prefer themes in native i3 config syntax you should check out [j4-make-config](http://github.com/okraits/j4-make-config).

**Comment by djornada** [comment]
i mean support alias to generate theme, when the config file contains alias

**Comment by okraits** [comment]
[j4-make-config](http://github.com/okraits/j4-make-config) does that.

**Comment by acrisci** [comment]
> j4-make-config does that.

Yes, if you don't mind using a far inferior config templating thing for i3!

Anyway, I'm guessing you mean to support any single word beginning with `$` to be inserted into the correct place in the config. This would be ok with me to put in. Patches welcome.

I would like to leave room for rgb conversions as well, so I would be wary of supporting any arbitrary string in the config file.

**Comment by okraits** [comment]
[j4-make-config](http://github.com/okraits/j4-make-config) simply replaces 2 markers in an i3 base config file with 2 blocks of i3 syntax for window and i3bar themeing read from a theme file. It uses native i3 syntax and python which is installed on nearly every linux desktop system out there.

You're welcome to explain why adding another markup language and another interpreter to the requirements for i3 themeing is a superior approach ;-)
